public/app/core/components/TraceToLogs/TraceToLogsSettings.tsx

```diff
@@ -26,8 +25,42 @@ export interface TraceToLogsOptions {
   lokiSearch?: boolean; // legacy
 }
 
+export interface TraceToLogsOptionsV2 {
```

Comment:
I opted for a totally new attribute in the JSON data so it is easier to figure out which version is used. Also makes it easier to just transform the old data and save as new version.